docs/install.md

```diff
@@ -57,24 +30,28 @@ bash Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
 
 (or choose any other path)
 
-#### 3. Install phy and its dependencies
+On Windows, launch the GUI Miniconda installer.
+
+#### 3. Install dependencies
 
 ```bash
-conda install conda python=3 pip numpy matplotlib scipy h5py pyqt ipython-notebook requests && pip install vispy phy
+conda install conda python=3 pip numpy matplotlib scipy h5py pyqt ipython-notebook requests --yes && conda install -c http://conda.binstar.org/kwikteam klustakwik2 --yes && pip install vispy
 ```
 
-On Windows, install the binary klustakwik2 package:
+#### 4. Install phy
+
+To install the latest stable version:
+
 ```
-conda install -c http://conda.binstar.org/kwikteam klustakwik2
+pip install phy
 ```
 
-On Mac / Linux (or if you want to compile from source in Windows)
+To install the development version (you must have **git** installed, for example using the GitHub desktop client):
+
 ```
-pip install klustakwik2
+git clone git://github.com/kwikteam/phy.git && cd phy && python setup.py develop
```

Comment:
where does this store the git file? is this properly tested? will doing it again update the git?